### ISSUE TITLE
Chamfer Distance Loss Values

### ISSUE THREAD

**Issue by rjtshrm** [Tong-ZHAO/Pixel2Mesh-Pytorch#13]
I used your chamfer distance loss in my model.  The minimum chamfer distance loss value of all epochs is 387.50 (batch_size = 30, n_epochs = 100, total_dataset_size = 3600, trainloader_len = 120 (total_dataset_size / total_dataset_size )). Is it optimal value for loss or should it be very low like 0.007 (just example). Each Point cloud has 4096 points (both for predicted and target).  Below is also the code for calculating the loss over each epoch

```
start_epoch = 0
n_epochs = 100
for epoch in range(start_epoch, n_epochs):
    running_loss = 0.0
    for i, data in enumerate(trainloader):
        # data is tuple consists of inputs and targets
        inputs, targets = data

        # make gradient zeros
        optimizer.zero_grad()
        
        # forward + backward + optimize
        predicted_outputs = net(inputs)

        dist1, dist2 = chamfer_dist(targets.permute(0, 2, 1).float(), predicted_outputs.permute(0, 2, 1).float())

        loss = (torch.mean(dist1)) + (torch.mean(dist2))

        loss.backward()
        optimizer.step()
        
        running_loss += loss.item()
        
    print("EPOCH {0}, LOSS {1}".format(epoch, running_loss / trainloader_len))
```
